src/llamafactory/model/model_utils/visual.py

```diff
@@ -161,6 +161,22 @@ def get_image_seqlen(config: "PretrainedConfig") -> int:
     return image_seqlen
 
 
+def get_patch_size(config: "PretrainedConfig") -> int:
+    r"""
+    Computes the patch size of the vit.
+    """
+    patch_size = getattr(config.vision_config, "patch_size", 14)
```

Comment:
maybe it is better to set the default value to -1?

Reply:
Yeah, -1 can let people to set a right one for some cases